docs/notifications/getting-started.md

```diff
@@ -0,0 +1,220 @@
+
+Depending on the needs, a processor can modify the content of a notification or route it to different systems like email, Slack, or other services.
+
+A good example of how to write a processor is the [Email Processor](https://github.com/backstage/backstage/tree/master/plugins/notifications-backend-module-email).
+
+Start off by creating a notification processor:
+
+```ts
+import { Notification } from '@backstage/plugin-notifications-common';
+import { NotificationProcessor } from '@backstage/plugin-notifications-node';
+
+class MyNotificationProcessor implements NotificationProcessor {
+  async decorate(notification: Notification): Promise<Notification> {
+    if (notification.origin === 'plugin-my-plugin') {
+      notification.payload.icon = 'my-icon';
+    }
+    return notification;
+  }
+
+  async send(notification: Notification): Promise<void> {
+    nodemailer.sendEmail({
+      from: 'backstage',
+      to: 'user',
+      subject: notification.payload.title,
+      text: notification.payload.description,
+    });
+  }
+}
+```
+
+Both of the processing functions are optional, and you can implement only one of them.
+
+Add the notification processor to the notification system by:
+
+```ts
+import { notificationsProcessingExtensionPoint } from '@backstage/plugin-notifications-node';
+import { Notification } from '@backstage/plugin-notifications-common';
+
+export const myPlugin = createBackendPlugin({
+  pluginId: 'myPlugin',
+  register(env) {
+    env.registerInit({
+      deps: {
+        notifications: notificationsProcessingExtensionPoint,
+        // ...
+      },
+      async init({ notifications }) {
+        // ...
+        notifications.addProcessor(new MyNotificationProcessor());
+      },
+    });
+  },
+});
+```
+
+### External Services
+
+When the emitter of a notification is a Backstage backend plugin, it is mandatory to use the integration via `@backstage/plugin-notifications-node` as described above.
+
+If the emitter is a service external to Backstage, an HTTP POST request can be issued directly to the API, assuming that authentication is properly configured.
+Refer to the [service-to-service auth documentation](https://backstage.io/docs/auth/service-to-service-auth) for more details, focusing on the Static Tokens section for the simplest setup option.
+
+An example request for creating a broadcast notification might look like:
+
+```bash
+curl -X POST https://[BACKSTAGE_BACKEND]/api/notifications -H "Content-Type: application/json" -H "Authorization: Bearer YOUR_BASE64_SHARED_KEY_TOKEN" -d '{"recipients":{"type":"broadcast"},"payload": {"title": "Title of broadcast message","link": "http://foo.com/bar","severity": "high","topic": "The topic"}}'
+```
+
+## Additional info
+
+Additional details can be found in the plugins' implementation:
```

Comment:
Would it make sense to include any existing implementations? I know we added an action to the Scaffolder that sends a notification - https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend-module-notifications. If this gets merged it would be a good example to point to for signals: https://github.com/backstage/backstage/pull/23194